docs/syncAllTemplates.md

```diff
@@ -0,0 +1,20 @@
+## How to Run the `syncAllTemplates.mjs` Script
+
+To run the `syncAllTemplates.mjs` script, you will need to add a new node script into the linked component and point that to the `syncAllTemplates.mjs` file. You can individually run the workflow configurations by pointing to the `syncAllTemplates.mjs` file and adding a `--github` parameter after the path. The same can be done for the linter configurations by adding a `--linters` parameter.
+
+### Example Calls
+
+```
+// Default
+"syncTemplates": "./node_modules/@aurodesignsystem/auro-library/scripts/setup/syncAllTemplates.mjs"
+```
+
+```
+// Only sync github workflow templates
+"syncTemplates": "./node_modules/@aurodesignsystem/auro-library/scripts/setup/syncAllTemplates.mjs --github"
+```
+
+```
+// Only sync linter configuration templates
+"syncTemplates": "./node_modules/@aurodesignsystem/auro-library/scripts/setup/syncAllTemplates.mjs --linters"
+```
```

Comment:
Please either have this in the README or there is a reference to this document in the README. The `./docs` directory is mainly used for secondary information or long descriptions of repo documentation. 

https://en.wikipedia.org/wiki/README

A README file typically encompasses:
- Configuration instructions
- Installation instructions
- **Operating instructions**
- A file manifest (a list of files in the directory or archive)
- Copyright and licensing information
- Contact information for the distributor or author
- A list of known bugs[3]
- Troubleshooting instructions[3]
- Credits and acknowledgments
- A changelog (usually aimed at fellow programmers)
- A news section (usually aimed at end users)